api/src/utils.js

```diff
@@ -18,6 +18,16 @@ function validatePassword(password) {
   return schema.validate(password);
 }
 
+function generatePassword() {
+  let length = 6;
+  let charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+  let retVal = "";
+  for (let i = 0, n = charset.length; i < length; ++i) {
+    retVal += charset.charAt(Math.floor(Math.random() * n));
+  }
+  return retVal;
+}
```

Comment:
Si j'ai bien compris on génère un mot de passe "qui ne sert à rien". Dans ce cas pourquoi on ne fait pas un mot de passe securisé (crypto OK) contrairement à Math.random qui est prédictible et qui ne prend que des lettres majuscules ?

Tant qu'à faire on peut réutiliser : 

```suggestion
function generatePassword() {
  return crypto.randomBytes(20).toString("hex");
}
```